resources/scss/bootstrap/_pagination.scss

```diff
@@ -1,9 +1,14 @@
 .pagination {
```

Comment:
I noticed that there was a bunch of code missing in there. Anyway I agree that bootstrap should be kept vanilla.